packages/auth/src/shared/gnapErrors.ts

```diff
@@ -19,11 +19,53 @@ export interface GNAPErrorResponse {
   }
 }
 
-export function throwGNAPError(
-  ctx: ExtendableContext,
-  httpCode: number,
-  gnapCode: GNAPErrorCode,
-  description?: string
-): never {
-  ctx.throw(httpCode, gnapCode, { error: { code: gnapCode, description } })
+export class GNAPServerRouteError extends Error {
+  public status: number
+  public code: GNAPErrorCode
+
+  constructor(status: number, code: GNAPErrorCode, message?: string) {
+    super(message)
+    this.status = status
+    this.code = code
+  }
+}
+
+export async function gnapServerErrorMiddleware(
```

Comment:
We should also check for `OpenAPIValidatorMiddlewareError` here as well, similar to `backend`, so we get a nicely formatted error when getting validation errors as well. This means we need to update `@interledger/openapi` to 2.0.1